pages/DSNP/Types/ProfileResource.md

```diff
@@ -0,0 +1,58 @@
+# CID
+
+A profile-linked resource is represented by the ProfileResource object type.
+
+## Serialization
+
+ProfileResource object serialization MUST conform to the following [Avro](https://avro.apache.org) schema:
+
+```
+{
+    "namespace": "org.dsnp",
+    "name": "ProfileResource",
+    "type": "record",
+    "doc": "Profile-linked resource",
+    "fields": [
+        {
+            "name": "type",
+            "type": "int",
+            "doc": "Type of resource"
+        },
+        {
+            "name": "cid",
+            "type": "string",
```

Comment:
Is a string the best way to represent a CID in Avro?

Either way we should be more clear how to serialize a CID into a string.

Specifically this should be a `cidv1` I believe